lib/carraway/server.rb

```diff
@@ -6,11 +6,17 @@
 
 module Carraway
   class Server < Sinatra::Base
-    set :views, ::File.expand_path('../views', __FILE__)
     set :method_override, true
     enable :sessions
     use Rack::Flash
 
+    def view(template)
+      if Config.views && ::File.exists?("#{Config.views}/#{template}.erb")
+        return erb template, views: Config.views
+      end
+      erb template
```

Comment:
Does this code path take a correct template path without `views` option?

Reply:
Yes.
There is no problem because sinatra is set by default.
https://github.com/sinatra/sinatra/blob/2e368238238358538cc0e02fe900f3ea05705c0a/lib/sinatra/base.rb#L1828-L1829